Userland/Libraries/LibAudio/MP3Tables.h

```diff
@@ -388,4 +388,14 @@ AK::Array<double, 512> WindowSynthesis {
     0.000030518, 0.000030518, 0.000015259, 0.000015259, 0.000015259, 0.000015259, 0.000015259, 0.000015259
 };
 
+auto SynthesisSubbandFilterCoefficients = []() {
+    AK::Array<AK::Array<double, 32>, 64> result;
+    for (size_t i = 0; i < 64; i++) {
+        for (size_t k = 0; k < 32; k++) {
+            result[i][k] = AK::cos((16 + i) * (2 * k + 1) * AK::Pi<double> / 64.0);
+        }
+    }
+    return result;
+}();
```

Comment:
Can this be written with constexpr?

Reply:
const-initing arrays is difficult
I would recommend getting all the values with a python script and pasting them in
(the script should be left as a comment in the code)

Reply:
That's probably best

Reply:
const initializing `AK::Array`s is very possible, check out `DeflateTables.h` for examples.